PubNub/Network/Parsers/PNSubscribeParser.m

```diff
@@ -225,14 +221,14 @@ + (NSMutableDictionary *)eventFromData:(id)data forChannel:(NSString *)channel
         [event addEntriesFromDictionary:[self messageFromData:data
```

Comment:
@jzucker2 Please pass _data[kPNMessagePayloadDataKey]_ instead of hole event envelop (_data_). This method don't care about outer data structure and should work only with user published message. There is also message decryption inside of _-addEntriesFromDictionary:withAdditionalParserData:_ and it will fail from the start, because dictionary always will be passed to it (it know what there potentially encrypted data if it see string and some _additionalData_).